frontend/src/AlertConfiguration/AlertConfigurationComponents/AlertConfigurationControlPanel.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Button from '@material-ui/core/Button';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import Container from '@material-ui/core/Container';
+import CreateAlertConfiguration from './CreateAlertConfiguration'
+
+const useStyles = makeStyles((theme) => ({
+  heroContent: {
+    backgroundColor: theme.palette.background.paper,
+    padding: theme.spacing(8, 0, 6),
+    marginBottom: 30,
+  },
+}));
+
+export default function AlertConfigurationControlPanel(props) {
+  const classes = useStyles();
+
+  var [clicked, setClick] = React.useState(false);
+
+  function handleClick() {
+    setClick(true);
+  }
+
+  if (!clicked) {
+    return(
+      <main>
+        <Container className={classes.heroContent}>
+          <Typography component="h1" variant="h2" align="center" color="textPrimary" gutterBottom>
+            Alert Configuration
+          </Typography>
+          <Typography variant="h5" align="center" color="textSecondary" paragraph>
+            Create custom configurations for GreySwan alerts
+          </Typography>        
+          <div>
+            <Grid container spacing={2} justify="center">
+              <Grid item>
+                <Button onClick={() => handleClick()} variant="contained" color="primary">
+                  Create
+                </Button>
+              </Grid>
+              <Grid item>
+                <Button variant="outlined" color="primary">
+                  Delete
+                </Button>
+              </Grid>
+            </Grid>
+          </div>
+        </Container>
+        {props.configurations}
+        {clicked = false}
+      </main>
+    );
+  } else {
+    return(
+      <main>
+        <CreateAlertConfiguration/>
+        {clicked = false}
+      </main>
```

Comment:
so my understanding of how this works now is that there's one state "clicked" for the Create button, and the page will render the Create form instead of the page of configs when it is clicked on. how will this work with the Delete button? do you plan on having another state for handling the Delete button being clicked on?